src/main/java/com/mercadolibre/be_java_hisp_w31_g3/repository/IUserRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.mercadolibre.be_java_hisp_w31_g3.repository;
+
+import com.mercadolibre.be_java_hisp_w31_g3.model.User;
+
+public interface IUserRepository {
+    User getUserById(Long userId);
```

Comment:
```suggestion
    User getById(Long userId);
```

porque ya sabemos que al estar en el repositorio del User lo que vamos a traer por Id es el User